### PR TITLE
Login, register authentication

### PR DESCRIPTION
**Pull request by KarolinaZielonka**

### CODE REVIEW

File: src/services/auth.service.ts [hunk 1]
```diff
@@ -0,0 +1,39 @@
+import axios from "axios";
+
+const API_URL = "http://localhost:3000/api/auth/";
```
Comment (rafalsz98) [inline src/services/auth.service.ts:3]: Default port of Spring backend is 8080

File: src/services/auth.service.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+class AuthService {
```
Comment (rafalsz98) [inline src/services/auth.service.ts:5]: @TomekStaszkiewicz can we use classes?

Comment (TomekStaszkiewicz) [reply src/services/auth.service.ts:5]: Yes, I think that we can for cases like this

Comment (TomekStaszkiewicz) [reply src/services/auth.service.ts:5]: It will be more readable for people who are not so familiar with js/ts

**Review by rafalsz98 — APPROVED** [review]

**Pull request merged**